docker-compose.yml

```diff
@@ -303,13 +303,16 @@ services:
     image: us.gcr.io/galoy-org/bria:edge
     environment:
       - PG_CON=postgres://user:password@postgres-bria:5432/pg
-    command: bash -c "
-      bria daemon \
-      --config bria.yml \
-      dev \
-      -x tpubDD4vFnWuTMEcZiaaZPgvzeGyMzWe6qHW8gALk5Md9kutDvtdDjYFwzauEFFRHgov8pAwup5jX88j5YFyiACsPf3pqn5hBjvuTLRAseaJ6b4
-      -d m/84h/0h/0h"
-    working_dir: /repo/dev
```

Comment:
@vindard this was the issue - the release image is hardened to a specific user that only has access to certain paths in the image.